deploy/srep-1000-ethtool-exporter/09-DaemonSet.yaml

```diff
@@ -0,0 +1,190 @@
+            - --path.udev.data="/host/root/run/udev/data"
+            - --collector.disable-defaults
+            - --collector.ethtool
+            - --collector.ethtool.metrics-include=^(bw_in_allowance_exceeded|bw_out_allowance_exceeded|conntrack_allowance_exceeded|linklocal_allowance_exceeded|pps_allowance_exceeded)$
+            - --runtime.gomaxprocs=0
+          name: ethtool-exporter
+          image: prom/node-exporter
+          command:
+            - /bin/sh
+            - -c
+            - |
+              export GOMAXPROCS=4
+              # We don't take CPU affinity into account as the container doesn't have integer CPU requests.
+              # In case of error, fallback to the default value.
+              NUM_CPUS=$(grep -c '^processor' "/proc/cpuinfo" 2>/dev/null || echo "0")
+              if [ "$NUM_CPUS" -lt "$GOMAXPROCS" ]; then
+                export GOMAXPROCS="$NUM_CPUS"
+              fi
+              echo "num_cpus=$NUM_CPUS gomaxprocs=$GOMAXPROCS"
+              exec /bin/node_exporter "$0" "$@"
+          env:
+            - name: DBUS_SYSTEM_BUS_ADDRESS
+              value: unix:path=/host/root/var/run/dbus/system_bus_socket
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 250m
+              memory: 180Mi
+            requests:
+              cpu: 8m
+              memory: 32Mi
+          securityContext: {}
+          volumeMounts:
+            - name: rootfs
+              mountPath: /host/root
+              readOnly: true
+            - name: sysfs
+              mountPath: /host/sys
+              readOnly: true
+            - mountPath: /var/node_exporter/text/file
+              name: node-exporter-textfile
+              readOnly: true
+            - name: procfs
+              mountPath: /host/proc
+              readOnly: true
+          workingDir: /var/node_exporter/textfile
+        - args:
+            - --secure-listen-address=[$(IP)]:9500
+            - --tls-cipher-suites=TLS_AES_128_GCM_SHA256,TLS_AES_256_GCM_SHA384,TLS_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+            - --upstream=http://127.0.0.1:9500/
+            - --tls-cert-file=/etc/tls/private/tls.crt
+            - --config-file=/etc/kube-rbac-policy/config.yaml
+            - --tls-min-version=VersionTLS12
+            - --tls-private-key-file=/etc/tls/private/tls.key
+            - --client-ca-file=/etc/tls/client/ca-bundle.crt
+          env:
+            - name: IP
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: status.podIP
+          image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:971174692a1aae68ac07ffa132254eb3c018a52507eb32621b6dc286fd25ca0d
```

Comment:
What version is this?

Reply:
Thanks for pointing this out Josh. The recent commit has been updated to add comments/info around each image + version. The images are taken from CMO on 4.16.42 MCs.  This would be better involving a trigger/stream, but not sure what makes sense for these images specifically?